docs/Using-Docker.md

```diff
@@ -26,13 +27,16 @@ Since Docker typically runs a container sharing a (linux) kernel with the host m
 Unity environment **has** to be built for the **linux platform**. When building a Unity environment, please select the following options from the the Build Settings window:
 - Set the _Target Platform_ to `Linux`
 - Set the _Architecture_ to `x86_64`
-- `Uncheck` the _Development Build_ option
-- `Check` the _Headless Mode_ option. (_This is required because the Unity binary will run in a container that does not have graphics drivers installed_.) 
 
-![Build Settings For Docker](images/docker_build_settings.png)
+![Build Settings For Docker](images/docker_build_settings_noheadless.png)
 
 Then click `Build`, pick an environment name (e.g. `3DBall`) and set the output directory to `unity-volume`. After building, ensure that the file `<environment-name>.x86_64` and subdirectory `<environment-name>_Data/` are created under `unity-volume`.
 
+**NOTE** If you are only collecting vector observations from Unity, you can select the `headless` option here:
+
+![Build Settings For Docker](images/docker_build_settings_headless.png)
```

Comment:
I don't see a difference between the *_headless.png and *_noheadless.png images, except that headless points to the GridWorld scene, while noheadless points to the 3DBall scene.

Reply:
That was a typo. I think I didn't commit the updated file. The `_headless.png` should be identical to the `_noheadless.png` except for the former having `headless` checked.